source/tutorials.rst

```diff
@@ -77,8 +93,11 @@ This is done by navigating to the ``$WECSIM/tutorials/rm3/hydroData/`` directory
 
 Step 2: Build Simulink Model
 """"""""""""""""""""""""""""""""""""""""""""""""
-
-The WEC-Sim Simulink model is created by dragging and dropping blocks from the *WEC-Sim Library* into the ``rm3.slx`` file. 
+.. Note:: 
+	Adam:
+	The simulink screenshots have illegibly small text. They need to be replaced. Make blocks smaller wrt text and zoom in for more clear figures. The base/follower note needs to come before a user connects blocks. Include why this difference is important.
+	
```

Comment:
I agree, we should definitely update these images and address the base/follower comment.